docs/user/troubleshooting-guides/03-00-cannot-connect-to-service/v1beta2/03-40-api-rule-troubleshooting.md

```diff
@@ -0,0 +1,162 @@
+```
+
+---
+## The **issuer** Configuration of **jwt** Access Strategy Is Missing
+### Cause
+
+The following APIRule is missing the **issuer** configuration for the **jwt** access strategy:
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      jwt:
+```
+
+If your APIRule is missing the **issuer** configuration for the **jwt** access strategy, the following error appears:
+
+```
+{"code":"ERROR","description":"Validation error: Attribute \".spec.rules[0].jwt\": supplied config cannot be empty"}
+```
+
+### Remedy
+
+Add JWT configuration for the **issuer** or ``. Here's an example of a valid configuration:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      jwt:
+        authentications:
+          - issuer: "https://dev.kyma.local"
+            jwksUri: "https://example.com/.well-known/jwks.json"
+```
+
+## Invalid **issuer** for the **jwt** Access Strategy
+### Cause
+
+Here's an example of the APIRule CR with an invalid **issuer** URL configured:
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      jwt:
+        authentications:
+          - issuer: ://unsecured.or.not.valid.url
+            jwksUri: https://example.com/.well-known/jwks.json
+```
+
+If the **issuer** contains `:`, it must be a valid URI. Otherwise, you get the following error, and the APIRule CR is not created:
+
+```
+The APIRule "httpbin" is invalid: .spec.rules[0].jwt.authentications[0].issuer: value is empty or not a valid url
+```
+
+### Remedy
+
+The JWT **issuer** must not be empty and must be a valid URI, for example:
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      jwt:
+        authentications:
+          - issuer: https://dev.kyma.local
+            jwksUri: https://dev.kyma.local/.well-known/jwks.json
+```
+
+---
+## Both **noAuth** and **jwt** Access Strategies Defined on the Same Path
+### Cause
+
+The following APIRule CR has both **noAuth** and **jwt** access strategies defined on the same path:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      noAuth: true
+      jwt:
+        authentications:
+          - issuer: https://dev.kyma.local
+            jwksUri: https://dev.kyma.local/.well-known/jwks.json
+```
+
+If you set **noAuth** access strategy to `true` and define the **jwt** configuration on the same path, you get the following error appears:
+
+```
+{"code":"ERROR","description":"Validation error: Attribute \".spec.rules[0].noAuth\": noAuth access strategy is not allowed in combination with other access strategies"}
+```
+
+### Remedy
+
+Decide on one configuration you want to use. You can either **noAuth** access to the specific path or restrict it using a JWT security token.
```

Comment:
Isn't a verb missing after `You can either`?

Reply:
it is, thanks